lib/stitches_norailtie.rb

```diff
@@ -14,7 +14,6 @@ def self.configuration
 require 'stitches/api_generator'
 require 'stitches/add_deprecation_generator'
 require 'stitches/add_enabled_to_api_clients_generator'
-require 'stitches/update_configuration_generator'
```

Comment:
I didn't see the purpose for leaving this in.  If anyone has not converted to the non-deprecated naming, they should run this generator before upgrading to 4.0, run their tests, deploy, and then do the 4.0 upgrade.

(I would put this information in the release notes instead of the README.)

Reply:
Happy to include in the release notes. Do you have any guidance on what those would be for this change?

Reply:
Ah, okay, sorry, I get it now. I thought this was related to the whitelist change it's self.